parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -366,6 +366,17 @@ ensureDHCPv6() {
 }
 
 ensureKubelet() {
+    KUBELET_DEFAULT_FILE=/etc/default/kubelet
+    mkdir -p /etc/default
+    echo "KUBELET_FLAGS=${KUBELET_FLAGS}" >> "${KUBELET_DEFAULT_FILE}"
```

Comment:
this actually duplicates all values in the file fyi because it's all appending. but it works since they just get sourced twice

Reply:
this has the cloud init wait before it right?

Reply:
yes it does, cloud init wait at the very beginning https://github.com/Azure/AgentBaker/blob/26dedd5c9270e0f77ed1303676ed31eb9e8ac114/parts/linux/cloud-init/artifacts/cse_cmd.sh#L2

and then later setting vars https://github.com/Azure/AgentBaker/blob/26dedd5c9270e0f77ed1303676ed31eb9e8ac114/parts/linux/cloud-init/artifacts/cse_cmd.sh#L133

Reply:
hmm, if I change the first redirect from >> to >, that solves the problem of duplication. @alexeldeib preference?

Reply:
updated PR to use > instead of >> to overwrite contents of /etc/default/kubelet to avoid duplication.